### DISCUSSION TITLE
How to cycle through multiple scrollable areas on a webpage

### DISCUSSION THREAD

**Discussion by lmdevv** [gdh1995/vimium-c#1229] [Q&A]
So as the question said, I want to scroll a webpage using j,k. However lets say the webpage has multiple scrollable areas, is there a way to cycle between this areas? How I tend to do it is using f and click the other scrollable area and then I can scroll that part, but is there a better way, I tried gf thinking that might be the way but it seems to do something else. What can I do? Thanks in advance

**Comment by gdh1995** [comment]
You may press `f` and then select a "dark red" mark, which usually means a scrollable area.

And there's `LinkHints.activateHover` to select page content without clicking it, so you may trigger it and select a link / button "inside the scrollable area that you want to scroll".
